hack/single-node/bootkube-up

```diff
@@ -9,12 +9,11 @@ fi
 
 # Render assets
 if [ ! -d "cluster" ]; then
-  ../../_output/bin/${local_os}/bootkube render --asset-dir=cluster --api-servers=https://172.17.4.100:443
+  ../../_output/bin/${local_os}/bootkube render --asset-dir=cluster $BOOTKUBE_RENDER_FLAGS
```

Comment:
Ah crap, I accidently included that. Thats my customization because bootkube's default stuff isn't very flexible. Let me remove that.